lambdas/check-existing-identity/src/main/java/uk/gov/di/ipv/core/checkexistingidentity/CheckExistingIdentityHandler.java

```diff
@@ -515,17 +516,40 @@ private JourneyResponse buildReuseResponse(
                         vcBundle.credentials, attainedVot.getProfileType()),
                 auditEventUser.getSessionId(),
                 false);
-        migrateCredentialsToEVCS(userId, vcBundle, evcsAccessToken);
+
+        if (!vcBundle.isEvcsIdentity) {
+            migrateCredentialsToEVCS(
+                    auditEventUser, deviceInformation, vcBundle.credentials, evcsAccessToken);
+        } else if (vcBundle.isF2fIdentity() && vcBundle.isPendingEvcsIdentity) {
+            updateIdentityOnF2fCompletion(
+                    auditEventUser.getUserId(), evcsAccessToken, vcBundle.credentials);
```

Comment:
This appears to duplicate @sam803 's work on F2F journeys - see https://github.com/govuk-one-login/ipv-core-back/pull/2033

I think you can just put the condition back into the condition below, and leave this PR as just adding the audit event.

Reply:
F2F completion update - before the above change migration call is taking care of this. After this change we are only migrating for only vc's not in evcs (!vcBundle.isEvcsIdentity). So f2f vc in pending_return in evcs will not get updated on completion.
Also in this case we don't want to call migration and raise migration event.

Reply:
Got it ...I think u mean now in case of pending we are doing new journey /reuse-with-store.  which will take care of that.

Reply:
Yep exactly :)